test/setup.ts

```diff
@@ -1,13 +1,6 @@
-import nock from 'nock';
-import 'jest-extended';
-
 jest.mock('../lib/platform', () => ({
   platform: jest.createMockFromModule('../lib/platform/github'),
   initPlatform: jest.fn(),
   getPlatformList: jest.fn(),
 }));
 jest.mock('../lib/logger');
-
-beforeAll(() => {
-  nock.disableNetConnect();
```

Comment:
I guess the idea of this was to block all external connections?

Reply:
yes, but it caused the memory issues, see https://github.com/nock/nock#memory-issues-with-jest